src/app/code/community/FireGento/MageSetup/Block/Bundle/Catalog/Product/Price.php

```diff
@@ -166,6 +163,9 @@ public function getIsShowWeightInfo()
      */
     public function getFormattedWeight()
     {
+        if ($this->getProduct()->isVirtual()) {
```

Comment:
```suggestion
        if ($this->getIsShowWeightInfo()) {
```

Wouldn't that even make more sense?

Reply:
you need to negate the if condition there

Reply:
True :see_no_evil: Fixed!